pytest.ini

```diff
@@ -31,3 +31,6 @@ norecursedirs =
 faulthandler_timeout = 480
 log_print = True
 log_level = INFO
+
+markers =
+    slow: slow test
```

Comment:
We are still discussing the scope of markers we want to use:
https://lists.apache.org/thread.html/4538437c96f599766005ba7829d0bee1511debb4f53599e0d300a56f%40%3Cdev.airflow.apache.org%3E